backend/src/main/java/ca/bc/gov/hlth/mohums/util/JwtTokenUtils.java

```diff
@@ -1,5 +1,6 @@
 package ca.bc.gov.hlth.mohums.util;
 
+import net.minidev.json.JSONArray;
```

Comment:
Delete format changes and un-used import (unless this import addition fixed something)